website/src/index.css

```diff
@@ -21,20 +21,38 @@ limitations under the License.
   box-sizing: border-box;
   margin: 0;
   padding: 0;
-  background-color: #1F1D1D;
+  
   font-family: 'Rubik', sans-serif;
   
 }
+body{
+  background-color: #1F1D1D;
+}
 
 
 
 /*Basic class*/
 
 .flex,
-.justify--content{
+.justify--content,
+.space--between{
   display: flex;
 }
 
 .justify--content{
   justify-content: center;
+}
+.space--between{
+  justify-content: space-between;
+}
+
+.line{
+  border: 1px solid #B5ADAD;
+  margin-top: 100px;
+  margin-bottom: 100px;
+}
+
+.line2{
+  border: 1px solid #B5ADAD;
+  margin-top: 10px;
```

Comment:
`line2` is not a very self-informative name, especially when there is a `line` class right above. You should perhaps add a comment to explain how  `line` and `line2` are different